packages/opentelemetry-node/docs/intro.asciidoc

```diff
@@ -0,0 +1,52 @@
+////
+Goal of this doc:
+Provide all the information a user needs to determine if the product is a good enough fit for their use case to merit further exploration
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+////
+
+[[intro]]
+== Elastic Distribution for OpenTelemetry Node.js
+
+++++
+<titleabbrev>Introduction</titleabbrev>
+++++
+
+:language: Node.js
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+////
+✅ Intro
+////
+The {distro_name} ("the distro") is a {language} package that provides:
+
+* An easy way to instrument your application with OpenTelemetry.
+* Configuration defaults for best usage.
+
+////
+✅ What is it?
+✅ Why use it?
+////
+A _distribution_ is a wrapper around an upstream OpenTelemetry repository with some customizations. The {distro_name} is a wrapper around the https://opentelemetry.io/docs/languages/js[OpenTelemetry SDK for {language}] and includes the following customizations:
+
+* Uses Elastic-specific processors that ensure optimal compatibility when exporting OpenTelemetry signal data to an Elastic backend like Elastic APM server.
+* Preconfigures the collection of tracing and metrics signals, applying some opinionated defaults, such as which sources are collected by default.
+* Ensures that the OpenTelemetry protocol (OTLP) exporter is enabled by default.
```

Comment:
TODO: This list should be refined. 
- Currently we don't have any `Elastic-specific processors ...` in the Node.js distro. It is possible processors are using the .NET or Java distros.
- "Ensures that the OpenTelemetry protocol (OTLP) exporter is enabled by default." ... IIUC OTLP **is** used by default with the vanilla OpenTelemetry SDK already, so this isn't really a customization.

Did these bullets come from one of the other distros? If so, I could compare/ask there.

Reply:
>This list should be refined.

My intention with these bullet points is to explain what differentiates the Elastic distro from [OpenTelemetry Node SDK](https://github.com/open-telemetry/opentelemetry-js/tree/main/experimental/packages/opentelemetry-sdk-node) (it's unclear what the exact name of this is 🙃) and why a user should choose the Elastic distro. For the Node distro, should I pull inspiration from [this list](https://github.com/elastic/elastic-otel-node/tree/main/packages/opentelemetry-node#why-this-distribution)?

Reply:
> (it's unclear what the exact name of this is

"OpenTelemetry Node.js SDK" should work.



> For the Node distro, should I pull inspiration from [this list](https://github.com/elastic/elastic-otel-node/tree/main/packages/opentelemetry-node#why-this-distribution)?

Yes, that is probably the best current list, though I will grant that I'm not a marketer.  Far and away the primary argument for our distro is to improve the developer experience, mostly in the "getting started" for the common case.

Reply:
>"OpenTelemetry Node.js SDK" should work.

Thanks! 

>though I will grant that I'm not a marketer

I'm definitely not either! I just like to add a dash 🧂  of "why should i use this" in the intro of the docs to help the reader gain confidence that they're making the right choice before they actually download/install a thing.

Reply:
> "OpenTelemetry Node.js SDK" should work.

Or, I guess I also used "OpenTelemetry SDK for Node.js" elsewhere. Both of those should be unambiguous for readers.